### DISCUSSION TITLE
Embedded notes

### DISCUSSION THREAD

**Discussion by antoniocanas** [SilentVoid13/Templater#748] [Help]
Is there any way to embed current daily note in a dashboard? I've tried:

![[<%+ tp.date.now() %>]] and ![[journal/<%+ tp.date.now() %>]]

but it didn't work

**Comment by scwunch** [comment]
You have an extra `+` in there.  Try this one: 
`![[<% tp.date.now() %>]]`

(and of course make sure the format matches your daily note format)

**Comment by scwunch** [comment]
Sorry, I had not yet learned about [Dynamic Commands](https://silentvoid13.github.io/Templater/docs/commands/dynamic-command/) when I wrote that response.

It seems, after playing with it for a bit more, that the `<%+` syntax does not actually put the result in the markdown file — only rendering it in the preview — and thusly [[wikilinks]] are not recognized by Obsidian as the links we intended.
![image](https://user-images.githubusercontent.com/7596442/138192470-09cb9509-6848-4e30-bc34-17bba1e0b53c.png)

Maybe there's a workaround?
